Userland/Applications/Browser/BookmarksBarWidget.cpp

```diff
@@ -1,117 +1,33 @@
 /*
- * Copyright (c) 2020, Emanuel Sprung <emanuel.sprung@gmail.com>
- * Copyright (c) 2022, networkException <networkexception@serenityos.org>
- * Copyright (c) 2023, Cameron Youell <cameronyouell@gmail.com>
+ * Copyright (c) 2024, the SerenityOS developers
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #include <Applications/Browser/BookmarksBarWidget.h>
 #include <Applications/Browser/Browser.h>
-#include <Applications/Browser/EditBookmarkGML.h>
+#include "EditBookmarkDialog.h"
```

Comment:
This should most likely be an include of the format `<Applications/Browser/...>`.